ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/EnginePayloadParameter.java

```diff
@@ -48,6 +48,9 @@ public class EnginePayloadParameter {
   private final List<String> transactions;
   private final List<WithdrawalParameter> withdrawals;
   private final List<DepositParameter> deposits;
+  private final List<Bytes32> versionedHashes;
+  private final Long dataGasUsed;
+  private final String excessDataGas;
```

Comment:
since they are both `QUANTITY` of 64 bits, it is better to uniform them using the same type for both

Reply:
Agreed, and I have a PR that unifies those. #5654 
@shemnon has some performance concerns, I have some readability concerns, and we think we can find a compromise. see convo here https://github.com/hyperledger/besu/pull/5536#discussion_r1247180442